src/providers/logs/logs.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Log } from '../../shared/models/log.model';
+import { UrlProvider } from '../url/url';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class LogsProvider {
+
+  constructor(public http: HttpClient, public urlProvider: UrlProvider) {}
+
+  public log = (log: Log): Observable<Object> => {
```

Comment:
This function is only used in the test now as the effect calls logMultiple() - is there a use where we would only send one log to the service , bypassing the store.

Reply:
I think we are going to have the SEND_LOGS action sending logs for us. And that works with multiple even if there is one log in the array. So I'll delete the `log` function. And I also going to rename `logMultiple` to `sendLogs` function.